internal/ls/utilities.go

```diff
@@ -390,7 +390,7 @@ func isInRightSideOfInternalImportEqualsDeclaration(node *ast.Node) bool {
 }
 
 func (l *LanguageService) createLspRangeFromNode(node *ast.Node, file *ast.SourceFile) *lsproto.Range {
-	return l.createLspRangeFromBounds(node.Pos(), node.End(), file)
+	return l.createLspRangeFromBounds(astnav.GetStartOfNode(node, file, false /*includeJSDoc*/), node.End(), file)
```

Comment:
This seems weird; do we actually want to do that for _all_ nodes? What happens if we need to make a range out of a JSDoc range itself?

Reply:
Yeah, that's why I changed it here.